Sming/Arch/Host/Components/hostlib/startup.cpp

```diff
@@ -271,6 +271,7 @@ int main(int argc, char* argv[])
 		return 1;
 	}
 
+	extern void host_init_bootloader();
```

Comment:
@mikee47  Can you move this line to the beginning of the file or a separate header file?